File: filesystem_local.lua
hg = require("harfang")

file = hg.Open('resources/pictures/owl.jpg')

if hg.IsValid(file) then
	print(string.format('File is %d bytes long',hg.GetSize(file)))
else
	print('Failed to open file')
end

hg.Close(file)

File: filesystem_local.go
package main

import (
	"fmt"

	hg "github.com/harfang3d/harfang-go/v3"
)

func main() {
	file := hg.Open("resources/pictures/owl.jpg")

	if hg.IsValid(file) {
		fmt.Printf("File is %d bytes long", hg.GetSize(file))
	} else {
		fmt.Printf("Failed to open file")
	}
	hg.Close(file)
}
